File: tests/practice_run_tests/python/solution_files/sol17.py
from __future__ import annotations 
from typing import Optional

class Solution:
  def solve(self, head: Optional[ListNode]) -> Optional[ListNode]:
    return head.next if head is not None else None
    
class ListNode:
  def __init__(self, val, next: Optional[ListNode]) -> None:
    self.val = val
    self.next = next

File: tests/practice_run_tests/java/solution_files/Sol17.java
package tests.practice_run_tests.java.solution_files;

public class Sol17 {
  public static ListNode solve(ListNode head) {
    return head != null ? head.next : null;
  }

  public static class ListNode {
    public int val;
    public ListNode next;

    public ListNode(int val, ListNode next) {
      this.val = val;
      this.next = next;
    }
  }
}